missing module xhistogram.xarray

I get a missing module when running the tests from a fresh environment:

FAILED wradlib/tests/test_dp.py::test_system_phidp - AttributeError: Module 'xhistogram.xarray' is not installed.

Thanks @egouden, normally you should get this error if `xhistogram` is missing:

```python
E       AttributeError: Module 'xhistogram.xarray' is not installed.
E       
E       You tried to access function/module/attribute 'histogram'
E       from module 'xhistogram.xarray'.
E       This module is optional right now in wradlib.
E       You need to separately install this dependency.
E       Please refer to https://docs.wradlib.org/en/stable/installation.html#optional-dependencies
E       for further instructions.

wradlib/util.py:74: AttributeError
```

But I missed to make the test depend on xhistogram and I missed to add it to the optional dependencies section. I'll have sent in a fix asap.

--- a/docs/installation.md
+++ b/docs/installation.md
@@ -148,6 +148,7 @@ As for now, the following dependencies are defined as optional:
 | netCDF4    | >= 1.0    | >= latest   |
 | requests   | >= 2.23.0 | >= latest   |
 | xmltodict  | >= 0.12   | >= latest   |
+| xhistogram | >= 0.3    | >= latest   |
 
 The following libraries are used by `netCDF4`, `h5py`/`h5netcdf` and `gdal` packages and should apply to these requirements:
 

--- a/docs/release_notes.md
+++ b/docs/release_notes.md
@@ -7,6 +7,13 @@ Please note that {{wradlib}} releases follow [semantic versioning](https://semve
 
 You can install the latest {{wradlib}} release from PyPI via ``$ python -m pip install wradlib`` or specific version via ``$ pip install wradlib==x.y.z``. The recommended installation process is described in {doc}`installation`.
 
+## Version 2.9.1
+
+**Bugfixes**
+
+* phidp histogram tests requre xhistogram.xarray  ({issue}`781`) reported by {at}`egouden`, {pull}`782`) by {at}`kmuehlbauer`
+
+
 ## Version 2.9.0
 
 **New features**

--- a/docs/release_notes.md
+++ b/docs/release_notes.md
@@ -11,7 +11,7 @@ You can install the latest {{wradlib}} release from PyPI via ``$ python -m pip i
 
 **Bugfixes**
 
-* phidp histogram tests requre xhistogram.xarray  ({issue}`781`) reported by {at}`egouden`, {pull}`782`) by {at}`kmuehlbauer`
+* phidp histogram tests require xhistogram.xarray  ({issue}`781`) reported by {at}`egouden`, ({pull}`782`) by {at}`kmuehlbauer`
 
 
 ## Version 2.9.0

--- a/wradlib/tests/__init__.py
+++ b/wradlib/tests/__init__.py
@@ -84,6 +84,7 @@ rioxarray = util.import_optional("rioxarray")
 pyproj = util.import_optional("pyproj")
 requests = util.import_optional("requests")
 xmltodict = util.import_optional("xmltodict")
+xhistogram = util.import_optional("xhistogram.xarray")
 
 
 requires_bottleneck = pytest.mark.skipif(
@@ -143,4 +144,9 @@ else:
 
 requires_geos = pytest.mark.skipif(
     not has_geos, reason="GDAL missing, or GDAL without GEOS"
+)
+
+requires_xhistogram = pytest.mark.skipif(
+    not util.has_import(xhistogram),
+    reason="requires xhistogram.xarray.",
 )

--- a/wradlib/tests/test_dp.py
+++ b/wradlib/tests/test_dp.py
@@ -11,6 +11,7 @@ import xarray as xr
 from scipy import integrate
 
 from wradlib import dp, util
+from wradlib.tests import requires_xhistogram
 
 
 @pytest.fixture(params=["lstsq", "cov", "matrix_inv", "lanczos_conv", "lanczos_dot"])
@@ -315,6 +316,7 @@ def test_unfold_phi_vulpiani_xarray(gamic_swp):
     gamic_swp.wrl.dp.unfold_phi_vulpiani(phidp="PHIDP", kdp="KDP")
 
 
+@requires_xhistogram
 def test_system_phidp():
     phidp = xr.DataArray(
         [

